examples/25-Chronos-2-examples.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>I would prefer if we showcased the example with <code>predict_likelihood_parameters=True</code> . Sampling with the "large" models can be quite computationally expensive, and only has benefits if auto-regression is used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2944/discussion/#comment-2531865942'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>